packages/react-core/src/components/Menu/examples/Menu.md

```diff
@@ -0,0 +1,738 @@
+      </Menu>
+    );
+  }
+}
+```
+
+### With links
+
+```js
+import React from 'react';
+import { Menu, MenuItem, MenuList, MenuListItem } from '@patternfly/react-core';
+
+class MenuWithLinks extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeItem: 0
+    };
+    this.onSelect = (event, itemId) => {
+      this.setState({
+        activeItem: itemId
+      });
+    };
+  }
+
+  render() {
+    const { activeItem } = this.state;
+    return (
+      <Menu onSelect={this.onSelect}>
+        <MenuList>
+          <MenuListItem isExternalLink to="#default-link1" itemId={0} isActive={activeItem === 0}>
+            Link 1
+          </MenuListItem>
+          <MenuListItem isExternalLink to="#default-link2" itemId={1} isActive={activeItem === 1}>
+            Link 2
+          </MenuListItem>
+          <MenuListItem to="#default-link3" itemId={2} isActive={activeItem === 2}>
+            Link 3
+          </MenuListItem>
+        </MenuList>
+      </Menu>
+    );
+  }
+}
+```
+
+### With separator(s)
+
+```js
+import React from 'react';
+import { Divider, Menu, MenuItem, MenuList, MenuListItem } from '@patternfly/react-core';
+
+class MenuWithSeparators extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeItem: 0
+    };
+    this.onSelect = (event, itemId) => {
+      this.setState({
+        activeItem: itemId
+      });
+    };
+  }
+
+  render() {
+    const { activeItem } = this.state;
+    return (
+      <Menu onSelect={this.onSelect}>
+        <MenuList>
+          <MenuListItem itemId={0} isActive={activeItem === 0}>
+            Action 1
+          </MenuListItem>
+          <MenuListItem itemId={1} isActive={activeItem === 1}>
+            Action 2
+          </MenuListItem>
+          <Divider component="li" />
+          <MenuListItem itemId={2} isActive={activeItem === 2}>
+            Action 3
+          </MenuListItem>
+        </MenuList>
+      </Menu>
+    );
+  }
+}
+```
+
+### With titled groups
+
+```js
+import React from 'react';
+import { Menu, MenuGroup, MenuItem, MenuList, MenuListItem, Divider } from '@patternfly/react-core';
+
+class MenuWithTitledGroups extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeItem: 0
+    };
+    this.onSelect = (event, itemId) => {
+      this.setState({
+        activeItem: itemId
+      });
+    };
+  }
+
+  render() {
+    const { activeItem } = this.state;
+    return (
+      <Menu onSelect={this.onSelect}>
+        <MenuGroup>
+          <MenuList>
+            <MenuListItem to="#default-link1" itemId={0} isActive={activeItem === 0}>
+              Link not in group
+            </MenuListItem>
+          </MenuList>
+        </MenuGroup>
+        <Divider />
+        <MenuGroup label="Group 1">
+          <MenuList>
+            <MenuListItem itemId={1} isActive={activeItem === 1}>
+              Link 1
```

Comment:
should these be labeled actions or updated with `to`?

Reply:
nice catch @mcoker - thank you